Jumoo.uSync.BackOffice/uSyncApplicationEventHandler.cs

```diff
@@ -120,7 +125,33 @@ private void Setup()
             }
         }
 
+		private void FileService_SavedTemplate(IFileService sender, global::Umbraco.Core.Events.SaveEventArgs<global::Umbraco.Core.Models.ITemplate> e) {
```

Comment:
Hi, the saved template events are handled within the TemplateHandler - this code probibly lives in the SavedEvent in the Handler - https://github.com/KevinJump/uSync/blob/Dev-v7_7/Jumoo.uSync.BackOffice/Handlers/TemplateHandler.cs#L142

Reply:
Hi KevinJump, thank you for the quick reply! But I don't quite understand, does this already handles files that are moved from root to sub directories?
With a clean install my issue is reproduced as following; 

1. Create doc type with template named 'myTest'
2. Move myTest.cshtml from root to ~/myTest in the Views folder
3. Move the doc type and template usync config files to a temporary location 
4. Remove myTest from UM (doc type and template)
5. Remove the uSyncActions.config file 
6. Move the uSync config files back to their original location
7. Hit the import button
8. uSync reports: 'Can't get template path (are template files missing?)'

Reply:
Hi

Yeah its not that this functionality works, it just that the File handler is the location for the Saved events etc... 

The code in the Template Serializer would be enough to get uSync looking in the sub folders, so it's just how you want the deletes and renames to be handled that will require changes on the events - but they are in the FileHandler, so best add them there.